Reason: automagic dependency on crywrap
Source: Daniel Mierswa <impulze@impulze.org>
Upstream: sent to mailinglist

From 06fe9dbf30ad47e87e260c9dd937fb1fd121d3a3 Mon Sep 17 00:00:00 2001
From: Daniel Mierswa <impulze@impulze.org>
Date: Sun, 15 Apr 2012 17:43:12 +0200
Subject: [PATCH] make crywrap optional

---
 configure.ac |    9 ++++++++-
 1 file changed, 8 insertions(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 5350a90..4ce677f 100644
--- a/configure.ac
+++ b/configure.ac
@@ -409,7 +409,14 @@ AC_CHECK_FUNCS([alarm atexit dup2 epoll_create kqueue memchr memset munmap \
 		putenv regcomp scandir select socket strcasecmp strchr \
 		strdup strerror strncasecmp strrchr strstr strtoul uname])
 
-PKG_CHECK_MODULES(LIBIDN, libidn >= 0.0.0, [libidn=yes], [libidn=no])
+AC_ARG_ENABLE(crywrap,
+	AS_HELP_STRING([--disable-crywrap], [unconditionally disable the crywrap TLS proxy service]))
+
+libidn=no
+
+if test "x$enable_crywrap" != "xno" ; then
+	PKG_CHECK_MODULES(LIBIDN, libidn >= 0.0.0, [libidn=yes], [libidn=no])
+fi
 
  if test "x$libidn" != "xno" && test "$ac_cv_func_daemon" != "no";then
   crywrap=yes
-- 
1.7.9.6

